### DISCUSSION TITLE
Detailview navigation missing

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15033] [Feature improvement]
It seems to be very inconvenient to move between records without navigation buttons in detail view . Or I missed something here?

Best

**Comment by waran70** [comment]
yetis dont care about this essential CRM function and this is proof that they are using excel to deal with customers, are they?
Before yetis gimme sour answer please do imagine you have really a lot of records what should be updated on daily basis. Cutting it out while forking vtiger makes users two or three clicks more.

**Reply by TonyM15** [reply]
Record list preview helps but limiting speed and visibility of whole dataset
